### PR TITLE
Add the Apache Pulsar helm chart repository

### PR DESCRIPTION
**Pull request by zzzming**

### CODE REVIEW

**Comment by mattfarina** [comment]
The [Artifact Hub](https://artifacthub.io) is another place you can list your repos. To do that, you login (or create an account), create an organization, and list your repository. You can link it to a user or organization.

File: repos.yaml [hunk 1]
```diff
@@ -1344,3 +1344,10 @@ repositories:
     maintainers:
       - name: Rasool Safari
         email: safari@rasooll.com
+  - name: pulsar
```
Comment (mattfarina) [inline repos.yaml:1347]: Why is this called `pulsar` instead of `kafkaesque`? Will Pulsar be the only thing in this repository?

Comment (zzzming) [reply repos.yaml:1347]: This is a Helm repo for Apache Pulsar, an open source data streaming and message queuing framework. We are the organization to support this Pulsar helm repo. Yes Pulsar will be the only thing in the given repo. Does this make sense?

**Comment by sijie** [comment]
@mattfarina I am one of the PMC members of Apache Pulsar. Pulsar team maintains an official project chart for each release. I have submitted a pull request #394 to add the official chart.

**Pull request merged**